skyportal/model_util.py

```diff
@@ -13,37 +13,69 @@
                               Source, Spectrum, Telescope, Thumbnail, User,
                               Token)
 
+def add_user(username, roles=[], auth=False):
+    user = User.query.filter(User.username == username).first()
+    if user is None:
+        user = User(username=username)
+        if auth:
+            social = TornadoStorage.user.create_social_auth(
+                user,
+                user.username,
+                'google-oauth2'
+            )
 
-def add_super_user(username):
+    for rolename in roles:
+        role = Role.query.get(rolename)
+        if role not in user.roles:
+            user.roles.append(role)
+
+    DBSession().add(user)
+    DBSession().commit()
+
+    return User.query.filter(User.username == username).first()
+
+
+def make_super_user(username):
     """Initializes a super user with full permissions."""
     setup_permissions()  # make sure permissions already exist
-    super_user = User.query.filter(User.username == username).first()
-    if super_user is None:
-        super_user = User(username=username)
-        social = TornadoStorage.user.create_social_auth(super_user,
-                                                        super_user.username,
-                                                        'google-oauth2')
-    admin_role = Role.query.get('Super admin')
-    if admin_role not in super_user.roles:
-        super_user.roles.append(admin_role)
-    DBSession().add(super_user)
-    DBSession().commit()
+    add_user(username, roles=['Super admin'], auth=True)
+
+
+def provision_token():
+    """Provision an initial administrative token.
+    """
+    admin = add_user('admin@cesium-ml.org', roles=['Super admin'])
```

Comment:
I think "testadmin" is what's used in the new yaml-based seed user

Reply:
Or is this not meant to map onto that?

Reply:
It can be anything, I guess---it just has to be a user with enough permissions.